docs/notebooks/scalable_thompson_sampling_using_sparse_gaussian_processes.pct.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# %% [markdown]
+# # Scalable Thompson Sampling using Sparse Gaussian Process Models
+
+# %% [markdown]
+# In our other [Thompson sampling notebook](thompson_sampling.pct.py) we demonstrate how to perform batch optimization using a traditional implementation of Thompson sampling that samples exactly from an underlying Gaussian Process surrogate model. Unfortunately, this approach incurs a large computational overhead that scales polynomially with the optimization budget and so cannot be applied to settings with larger optimization budgets, e.g. those where large batches (>>10) of points can be collected.
+#
+# Luckily, Trieste also supports a scalable formulation of Thompson sampling. In particular, by coupling a sparse Gaussian process surrogate model with an approximate sampling scheme we can apply Thompson sampling over very large  optimization budgets and batch sizes, i.e. for those settings where standard BO methods are infeasbible. The implementation in this notebook follows [our paper](https://arxiv.org/pdf/2006.05356.pdf).
```

Comment:
Why not cite the paper same way we refer to all other publications in the docs?

Reply:
In notebooks we dont use the cite thing I think. Not sure if it would work? I think this is cleaner for notebooks

Reply:
I think we do. Have a look this paragraph of the GIBBON notebook: https://secondmind-labs.github.io/trieste/notebooks/batch_optimization.html#Batch-acquisition-functions.

Reply:
In notebooks we dont use the cite thing I think. Not sure if it would work? I think this is cleaner for notebooks

Reply:
yep, we do use it